packages/dx-react-chart-material-ui/src/templates/series/point.test.jsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { createShallow, getClasses } from 'material-ui/test-utils';
+import { Point } from './point';
+
+describe('Point', () => {
+  const defaultProps = {
+    x: 1,
+    y: 2,
+    d: 'M11 11',
+    style: {
+      strokeWidth: '1px',
+      fill: 'black',
+    },
+  };
+  const shallow = createShallow({ dive: true });
+  const classes = getClasses(<Point {...defaultProps} />);
+
+  it('should render path element', () => {
+    const tree = shallow((
+      <Point
+        {...defaultProps}
+      />
+    ));
+    expect(tree.find('path').exists()).toBeTruthy();
+  });
+
+  it('should render path element with props', () => {
+    const tree = shallow((
+      <Point {...defaultProps} />
+    ));
+    const { transform, d } = tree.find('path').props();
+    expect(transform).toBe('translate(1 2)');
+    expect(d).toBe('M11 11');
+  });
+
+  it('should render path element with custom styles', () => {
+    const customStyle = {
+      stroke: 'orange',
+      fill: 'green',
+    };
+    const tree = shallow((
+      <Point
+        {...defaultProps}
+        style={customStyle}
+      />
+    ));
+    const { style } = tree.find('path').props();
+    expect(style).toEqual({
+      ...customStyle,
+    });
+  });
+
+  it('should pass the className prop to the root element', () => {
+    const tree = shallow((<Point {...defaultProps} className="custom-class" />));
```

Comment:
Sometimes you use two brackets with `shallow`, sometimes only one.

`const tree = shallow(<Root {...defaultProps} customProperty>child</Root>);`

Reply:
Is it the [prettier](https://prettier.io/) time?